notes/stories/film/film-scifi-2020s.st.txt

```diff
@@ -443,6 +443,50 @@ standing up to a bully [In the end, Luke stood up to his domineering sister Mimi
 reconciliation [In the end, Luke stood up to his domineering sister and they hugged it out.]
 
 
+movie: Skylines (2020)
+======================
+
+:: Title
+Skylines
+
+:: Description
+A team of elite soldiers embark on a mission to a far off planet to
+exterminate the aliens there before those very aliens exterminate humanity. It
+is a sequel to Beyond Skyline (2017) and the final entry in the Skyline
+trilogy.
+
+:: Date
+2020-10-25
+
+:: References
+https://en.wikipedia.org/wiki/Skylines_(film)
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+attack from outer space by a genocidal enemy [A remnant of the human race launched a mission to the Harvester alien home world to kill them all before those same aliens returned to Earth to finish the job of "harvesting" all humans. From the Harvester aliens point of view, they humans had come to their home world to exterminate them.]
+human-alien hybrid [The heroine of the story Rose a human infused with Harvester alien DNA.]
+extraterrestrial being [The humans were battling aliens known as the Harvesters.]
+brother and sister [Central to the story was the relationship between Rose and her adoptive brother Trent.]
+human-made pandemic [In a surprise twist, it was revealed that General Radford was responsible for having released the virus that was decimating Earth's Pilot human-alien hybrid population.]
+genocide [A grand revelation of the film is that General Radford had deliberately released the virus in an effort to exterminate the Harvester aliens and peacefully euthanize the Pilot human-alien hybrids in one fell swoop. This was explicitly characterized as an act of genocide in the film.]
+
+:: Minor Themes
+freezing up in a critical moment [Rose froze before she could fire and one of her ships was destroyed as a result.]
+remorse [Rose was wracked with guilt over having sacrificed thousands of the fighters under her charge to destroy the Harvester alien mothership.]
+facing an epidemic [Dr. Mal was working to find a cure for the virus that was decimating Earth's Pilot human-alien hybrid population.]
+hologram [General Radford called up holographic maps of both Earth and the Harvester alien home world when briefing Rose on her mission.]
+speculative spaceship [Rose and her crew piloted a captured Harvester alien mothership to the Harvester alien home world.]
```

Comment:
You use this (and maybe a couple others) parent theme a lot and I keep thinking it would be nice if your motivation at least gave some example of what sort of attribute the spaceship had that made it speculative, and theme-worthy. I expect that would be easy enough.

Reply:
(tangentially, though I'm not complaining I note that the ownership of the ship and what it was used for is pretty irrelevant info)

Reply:
Motivation updated.